tests/wpt/web-platform-tests/dom/nodes/Document-createEvent.html

```diff
@@ -151,4 +151,8 @@
     }, 'Should throw NOT_SUPPORTED_ERR for pluralized non-legacy event interface "' + eventInterface + 's"');
   }
 });
+
+test(function() {
+  assert_equals(document.createEvent('storageevent'), "");
+}, "document.createEvent('storageevent') does not propagate the document's URL.")
```

Comment:
This is oddly specific for such a general test file. It would be more correct to test that for every event type that document.createEvent works, `var event2 = new event.constructor(e.type)` yields an event with matching properties (with the exception of `timeStamp` which is set every time an event is created).